Docker_Container/README.md

```diff
@@ -0,0 +1,54 @@
+## Steps to start the build with Docker 
+
+### Project Setup
+
+- Install [Docker](https://desktop.docker.com/win/stable/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&amp;utm_medium=webreferral&amp;utm_campaign=dd-smartbutton&amp;utm_location=module)
+
+- Build and Tag The Docker Image 
+
+``$ docker build -t sample:dev``
+
+- Then Spin up the container once build is done 
+
+``
```

Comment:
Minor style tweak - For multi line code blocks you have to use triple quoted backticks (`) , or  it'll be interpreted as a single line code block